### PR TITLE
TELCODOCS-950: Adding release note for networking pinning

### PR DESCRIPTION
**Pull request by rohennes**
[TELCODOCS-950](https://issues.redhat.com//browse/TELCODOCS-950): For networking workloads, you can dynaimically use non-reserved CPUS.

Version(s):
4.14

Issue:
https://issues.redhat.com/browse/TELCODOCS-950

Link to docs preview:
https://file.emea.redhat.com/rohennes/TELCODOCS-950-network-pinning/release_notes/ocp-4-14-release-notes.html#ocp-4-14-networking-kernal-network-pinning

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-14-release-notes.adoc [hunk 1]
```diff
@@ -72,6 +72,18 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-14-networking"]
 === Networking
 
+[id="ocp-4-14-networking-kernal-network-pinning"]
+==== Dynamic use of non-reserved CPUs for networking workloads
+
+With this release, networking workloads in performance-tuned clusters with a CPU manager policy set to `static` can dynamically use non-reserved CPUs by default.
```
Comment (MarSik) [inline release_notes/ocp-4-14-release-notes.adoc:78]: Ok, this needs a bit of rewrite. First it is not the workloads that will get access to the cpus, it is OVS (openvswitch). The workloads using OVS networking will benefit from it.

Basically replace Networking workloads with openvswitch or "OVN networking stack" and update the rest accordingly.

Comment (rohennes) [reply release_notes/ocp-4-14-release-notes.adoc]: @MarSik - thanks Martin, I updated the text. Hope this works.

**Comment by MarSik** [comment]
@zeeke Please check the networking terminology here. I think we usually use OVN and leave OVS as the implementation detail.

File: release_notes/ocp-4-14-release-notes.adoc [hunk 2]
```diff
@@ -72,6 +72,18 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-14-networking"]
 === Networking
 
+[id="ocp-4-14-networking-kernal-network-pinning"]
+==== Dynamic use of non-reserved CPUs for OVS
+
+With this release, the Open vSwitch (OVS) networking stack can dynamically use non-reserved CPUs by default in performance-tuned clusters with a CPU manager policy set to `static`. 
+This dynamic use of available, non-reserved CPUs maximizes compute resources for OVS and avoids network latency for workloads during periods of high demand.
```
Comment (ricky-rav) [inline release_notes/ocp-4-14-release-notes.adoc:79]: here instead of `avoids network latency` I'd say `minimizes network latency`. Other than this, it looks good to me!

File: release_notes/ocp-4-14-release-notes.adoc [hunk 3]
```diff
@@ -95,6 +95,18 @@ A new oc command line interface (CLI) flag, `--import-mode`, has been added to t
...
+[id="ocp-4-14-networking-kernal-network-pinning"]
+== Dynamic use of non-reserved CPUs for OVS
+
+With this release, the Open vSwitch (OVS) networking stack can dynamically use non-reserved CPUs by default in performance-tuned clusters with a CPU manager policy set to `static`. 
+This dynamic use of available, non-reserved CPUs maximizes compute resources for OVS and minimizes network latency for workloads during periods of high demand. 
+OVS remains unable to dynamically use isolated CPUs assigned to containers in `Guaranteed` QoS pods. This separation avoids disruption to critical application workloads. 
+
+[NOTE]
+====
+When the Node Tuning Operator (NTO) recognizes the performance conditions to activate the use of non-reserved CPUs, there is a few seconds delay while  `ovnkube-node` configures the CPU affinity alignment of OVS daemons running on the CPUs. During this window, if a `Guaranteed` QoS pod starts, it can experience a latency spike.
```
Comment (bergerhoffer) [inline release_notes/ocp-4-14-release-notes.adoc:107]: * Not critical, but I checked the Node Tuning Operator docs and it doesn't look like they really use the "NTO" abbreviation. And since you don't use it again in this section, I'd recommend removing it.
* "a few seconds delay" could probably be worded a bit better. Maybe "a several second delay"?
* Should it be clarified what `ovnkube-node` is?

```suggestion
When the Node Tuning Operator recognizes the performance conditions to activate the use of non-reserved CPUs, there is a several second delay while `ovnkube-node` configures the CPU affinity alignment of OVS daemons running on the CPUs. During this window, if a `Guaranteed` QoS pod starts, it can experience a latency spike. 
```

File: release_notes/ocp-4-14-release-notes.adoc [hunk 4]
```diff
@@ -95,6 +95,18 @@ A new oc command line interface (CLI) flag, `--import-mode`, has been added to t
 == Ingress Node Firewall Operator is generally available
 Ingress Node Firewall Operator was made a technology preview in {product-title} 4.12. With this release, Ingress Node Firewall Operator is generally available. You can now configure firewall rules at the node level. For more information, see xref:../networking/networking-operators-overview.adoc#networking-operators-overview[Ingress Node Firewall Operator].
 
+[id="ocp-4-14-networking-kernal-network-pinning"]
+== Dynamic use of non-reserved CPUs for OVS
+
+With this release, the Open vSwitch (OVS) networking stack can dynamically use non-reserved CPUs by default in performance-tuned clusters with a CPU manager policy set to `static`.
```
Comment (bergerhoffer) [inline release_notes/ocp-4-14-release-notes.adoc:101]: I know I'm not familiar with the material, but this sentence was hard to follow for me. I can't tell exactly what is new or what is the actual criteria for being able to dynamically used non-reserved CPUs by default. Do they have to be set to static to use this, or are ones set to static considered performance-tuned?

I might not be making much sense here, but maybe see if breaking it up into two sentences would be clearer?

**Review by bergerhoffer — COMMENTED** [review]
A few suggestions/questions!
